### PR TITLE
add farmland color

### PR DESCRIPTION
**Pull request by SeanConn15**
similar to openstreetmap.org's color, this adds farmland as a distinct color from other land areas.

I'm not sure which files I should be including here; so if I've included something I shouldn't have let me know.

Before:
https://imgur.com/a/zOrpRYZ
After:
https://imgur.com/a/CQFQfSH
night mode:
https://imgur.com/a/vsYWszq

related to https://github.com/organicmaps/organicmaps/issues/988

### CODE REVIEW

File: data/styles/clear/include/Basemap.mapcss [hunk 1]
```diff
@@ -299,6 +299,9 @@ area|z12-13[landuse=village_green],
 area|z12-13[landuse=field],
 {fill-color: @green0;}
 
+area|z14-[landuse=farmland],
+{fill-color: @farmland;fill-opacity: 1;}
```
Comment (vng) [inline data/styles/clear/include/Basemap.mapcss:303]: Remove useless entry here, you put it in separate category below.
And remove area|z14-[landuse=farmland], entry above in Ln 266

File: data/styles/clear/include/Basemap.mapcss [hunk 2]
```diff
@@ -569,6 +572,10 @@ area|z15-19[aeroway=aerodrome]
 area|z15-[landuse=churchyard]
 {fill-color: @background;}
 
+/* 7.8 Agricultural */
+area|z10-[landuse=farmland],
+{fill-color: @farmland;fill-opacity: 1;}
```
Comment (vng) [inline data/styles/clear/include/Basemap.mapcss:577]: Let's keep consistency here and draw farmlands starting from z12 like other similar landuses?

**Review by vng — CHANGES_REQUESTED** [review]

**Comment by vng** [comment]
Also, please, sign-off your commit like
git commit --amend --signoff

File: data/styles/clear/include/Basemap.mapcss (hunk 2 above)
Comment (SeanConn15) [reply data/styles/clear/include/Basemap.mapcss:577]: My thinking was to show them at a higher zoom level to give a better idea of land use similar to how we do with forests. I would call farmland more similar to forests than something like hospitals or fields; but I see the argument for either.

**Comment by SeanConn15** [comment]
Comments should be addressed; if @vng or someone else still wants to change the farmland layer I can do that as well.

**Comment by vng** [comment]
Ok good. @biodranik @rtsisyk colleagues, WDYT about first zoom level for farmlands? Leave it on 12 like before or make 10 like forests?

**Comment by biodranik** [comment]
Let's try 10, and see how it will look.

**Comment by vng** [comment]
Ok, I do approve, Linux CI is broken because of some other issues.

**Review by vng — APPROVED** [review]

**Review by biodranik — APPROVED** [review]

**Pull request merged**